### PR TITLE
Multiline support for logs statements in Filebeat

### PR DESCRIPTION
**Pull request by rafzei**

### CODE REVIEW

File: docs/home/howto/LOGGING.md [hunk 1]
```diff
@@ -74,3 +74,17 @@ Note: Currently input plugin doesn't officialy support skipping certificate vali
...
+By default postgresql block is provided, you can use it as example:  
+```yaml
+  postgresql_input:
+    multiline:
+      pattern: '^\d{4}-\d{2}-\d{2} '
+      negate: true
+      match: after
+```
+Currently supported inputs: `common_input`,`postgresql_input`,`docker_input`  
+More details about multiline options you can find in the official documentation: https://www.elastic.co/guide/en/beats/filebeat/current/multiline-examples.html
```
Comment (atsikham) [inline docs/home/howto/LOGGING.md:90]: IMO it's better to specify a hyperlink

**Review by atsikham — CHANGES_REQUESTED** [review]

File: docs/home/howto/LOGGING.md (hunk 1 above)
Comment (rafzei) [reply docs/home/howto/LOGGING.md:90]: That's true, thanks!

**Review by atsikham — DISMISSED** [review]
LGTM

**Review by mkyc — APPROVED** [review]
LGTM

**Review by atsikham — APPROVED** [review]

**Pull request merged**